src/extension.ts

```diff
@@ -6,20 +6,26 @@ import { StatusBarItem } from './statusBarItem'
 import { FlatpakRunner } from './flatpakRunner'
 import { TaskMode } from './taskMode'
 import { restoreRustAnalyzerConfigOverrides } from './integration/rustAnalyzer'
-import { findManifests } from './flatpakManifestUtils'
+import { findManifests, versionCompare } from './flatpakManifestUtils'
 import { FlatpakTerminal } from './flatpakTerminal'
+import { execSync } from 'child_process'
+import { Command } from './command'
 const { executeCommand, registerCommand } = commands
 
 export const EXT_ID = 'flatpak-vscode'
 
 // whether VSCode is installed in a sandbox
 export const IS_SANDBOXED = existsSync('/.flatpak-info')
-
+export let flatpakVersion: string
```

Comment:
It seems to be not imported with other files, it could be a local const variable

Reply:
Make sense, but then I thought it would be useful to stop executing commands in the future. Haven't thought of a clean way to do it yet

Reply:
there could be a helper function in utils.ts that can be tested which then called by `isValidVersion`